packages/react/src/components/navigation/navigation-row/NavigationRow.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Children, cloneElement, isValidElement, ReactElement, useContext, useEffect } from 'react';
+
+import styles from './NavigationRow.module.css';
+import classNames from '../../../utils/classNames';
+import NavigationContext from '../NavigationContext';
+import { NavigationRowDisplay } from '../Navigation.interface';
+
+export type NavigationRowProps = React.PropsWithChildren<{
+  /**
+   * Defines how the navigation row will be displayed.
+   * Supported values:
+   * fullWidth (default) - items will be displayed beneath the header
```

Comment:
`fullWidth` seems a bit misleading here. Not sure what would describe this option better though. "subMenu", "subNav", "subHeader"..?

Reply:
I agree. Changed it to `subNav`